be/src/util/system_metrics.h

```diff
@@ -40,6 +41,19 @@ class SystemMetrics {
     // update metrics
     void update();
 
+    void get_disks_io_time(std::map<std::string, int64_t>& map);
+    int64_t get_max_io_util(
+        const std::map<std::string, int64_t>& lst_value, int64_t interval_sec);
+
+    void get_network_traffic(
```

Comment:
output variable use pointer

Reply:
there is not output variables in parameter list